packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.js

```diff
@@ -80,6 +80,13 @@ export const BrowseStep = ({
   const assetCount = assets.length;
   const folderCount = folders.length;
 
+  const handleClickFolderCard = id => {
```

Comment:
Since we are only passing the arguments through, what do you think about spreading them?

```suggestion
  const handleClickFolderCard = (...args) => {
```